src/test/java/uk/ac/ebi/subs/utils/TestUtils.java

```diff
@@ -82,6 +83,9 @@ public static String getJWTToken (String authUrl, String username, String passwo
         HttpClient client = HttpClientBuilder.create().setDefaultCredentialsProvider(provider).build();
         HttpResponse response = client.execute(new HttpGet(authUrl));
 
+        if (response.getStatusLine().getStatusCode() != HttpStatus.SC_OK) {
```

Comment:
Should not we match the response with `HttpStatus. SC_UNAUTHORIZED` instead of not OK?
I might miss something.

Reply:
The tests should only run when we get the 200 OK. The other option is to have a series of `if` and `else` statements to account for all the different error codes.

Reply:
I understand that, but the message of the exception could be misleading.